src/alphafold_impact/pipelines/data_collection_oa/nodes.py

```diff
@@ -0,0 +1,117 @@
+import requests
+from tqdm import tqdm
+import math
+from typing import List
+import json
+from kedro_datasets.json import JSONDataset
+
+
+def retrieve_oa_works_for_single_concept_and_year(
+    concept_id: str, year: int, per_page: int
+) -> List[dict]:
+    """
+    Retrieves OpenAlex works for a single concept ID and publication year.
```

Comment:
It should be possible to include a list of concept_ids and request for id1 AND id2 AND id3 etc - if we end up with the full list (ie. to build a counterfactual of research done by those in the field), it may help greatly reduce redundancy in returned items.

Reply:
I've updated the function to accept a list of `concept_ids` now. There could still be duplicates retrieved due to the chunking in the main function.